Tribler/community/market/core/tick.py

```diff
@@ -62,8 +62,8 @@ def from_database(cls, data):
                         Timeout(timeout), Timestamp(timestamp), traded=traded, block_hash=str(block_hash))
 
     def to_database(self):
-        return (text_type(self.order_id.trader_id), int(self.order_id.order_number), self.assets.first.amount,
-                text_type(self.assets.first.asset_id), self.assets.second.amount,
+        return (database_blob(self.order_id.trader_id.to_string()), int(self.order_id.order_number),
```

Comment:
Casting trader_id to `database_blob` might be necessary in `to_database()` of the following files as well.
- community/market/core/payment.py
- community/market/core/order.py
- community/market/core/transaction.py

Reply:
Will do this later 👍